content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -1,99 +1,190 @@
+:guilabel:`Customer Preview` smart button, and the :guilabel:`Invoice` button reveals the drafted
+invoice. In either case, the down payment is mentioned on both the sales order and draft invoice.
+
+.. image:: down_payment/down-payment-line-item.png
    :align: center
-   :alt: What is a basic sales flow using down payments on Odoo Sales (3)?
+   :alt: How the down payment is mentioned on the sales order in Odoo Sales.
 
-This flow is also possible with a down payment taking into account a fixed amount.
+To confirm and register the first payment of the down payment, click the :guilabel:`Invoices` smart
+button to return to the invoice. On the :guilabel:`Customer Invoice` page, click
+:guilabel:`Confirm`, then :guilabel:`Register Payment` to open a :guilabel:`Register Payment`
+pop-up form.
+
+.. image:: down_payment/register-payment-popup.png
+   :align: center
+   :alt: How the register payment pop up looks in Odoo Sales.
+
+On this form, select the appropriate :guilabel:`Journal` and confirm that the down payment amount
+is correct in the :guilabel:`Amount` field. Then, make any other modifications to the other fields,
+if necessary, before clicking :guilabel:`Create Payment`.
+
+Odoo returns to the invoice, which now has a green `Paid` banner in the upper-right corner.
+
+.. image:: down_payment/paid-banner-invoice-1.png
+   :align: center
+   :alt: How the initial down payment invoice has a green paid banner in Odoo Sales.
+
+Now, when the customer wants to pay the remaining amount of the order, another invoice must be
+created. To do that, return to the sales order, and click :guilabel:`Create Invoice`.
+
+On the :guilabel:`Create invoices` pop-up, there is now an option to :guilabel:`Deduct down
+payments`, which is selected by default.
+
+.. image:: down_payment/deduct-down-payments-option.png
+   :align: center
+   :alt: The deduct down payment option on the create invoices pop up in Odoo Sales.
+
+If the remaining amount is ready to be paid, the :guilabel:`Regular Invoice` option should be
+selected, and the :guilabel:`Deduct down payments` option should *stay* selected. Then, click
+:guilabel:`Create and View Invoice`.
+
+Doing so reveals a separate :guilabel:`Invoices` page, listing all the invoices for that specific
+sales order. Each invoice line item displays all the necessary information related to each invoice,
+along with the current :guilabel:`Payment Status`.
+
+.. image:: down_payment/down-payment-invoices-page.png
+   :align: center
+   :alt: The down payment invoices page in Odoo Sales.
+
+To complete the flow, select the invoice with the :guilabel:`Not Paid` status, which opens that
+invoice. On the invoice page, click :guilabel:`Confirm`, then :guilabel:`Register Payment`, and
+confirm that the remaining :guilabel:`Amount` is correct. Then, click :guilabel:`Create Payment`.
+
+That pop-up disappears, and a green `Paid` banner is now present on the invoice. The line items on
```

Comment:
reminder: pop-up is a compound modifier in technical documentation, not a noun as we normally might refer to it as when speaking informally.

```suggestion
That pop-up window disappears, and a green `Paid` banner is now present on the invoice. The line items on
```